### ISSUE TITLE
Display OSD message when trying to increase volume when it's already maxed out

### ISSUE THREAD

**Issue by pozmu** [malensek/3RVX#75]
Hi
Thanks for great app.
Could you please add new feature that would show volume bar when user is trying to increase volume when it's already on max, so there would be some feedback? It could be shown in other color or with message saying "max" or something like that. 
Thanks

**Comment by dgw** [comment]
Yup. macOS shows the system OSD (and plays the sound effect) even if the volume is already at max, to give the user feedback that they can't boost it any more.
